browser/js/common/directives/reviews/reviews.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+app.directive('reviewPanel', function (SingleItemFactory){
+	return {
+		restrict: 'E',
+		templateUrl: 'js/common/directives/reviews/reviewPanel.html',
+		link: function (scope, elem, attr){
+			scope.submitReview = function (review, itemid, userid){
```

Comment:
inject CurrentFactory so that you don't need to pass in userid. Userid should always be current user, and you can pull that information from the factory.